.github/workflows/pr.yaml

```diff
@@ -126,6 +128,6 @@ jobs:
         run: |
           pnpm scripts:setup
 
-      - name: Run test coverage
+      - name: Run test
```

Comment:
Why we are changing to just test?

Reply:
because test coverage is broken, I think we need to fix it and put coverage later